modules/backend/behaviors/relationcontroller/partials/_manage_form.htm

```diff
@@ -47,7 +47,13 @@ <h4 class="modal-title">
 
     <?php else: ?>
 
-        <?= Form::ajax('onRelationManageCreate', ['data-popup-load-indicator' => true, 'sessionKey' => $newSessionKey]) ?>
+        <?= Form::ajax('onRelationManageCreate', [
+            'data-popup-load-indicator' => true,
+            'data-request-success' => "
+                $.oc.relationBehavior.changed('" . e($this->vars['relationField']) . "', 'created')
```

Comment:
make this inline please

Reply:
Done in 46d1e0f1611ceee10cd5d920e2690fa8b62465b5.